### DISCUSSION TITLE
How to handle switching windows confirmation box "Did you mean to switch apps?"?

### DISCUSSION THREAD

**Discussion by rahulmishra1991** [microsoft/react-native-windows#12244] [Q&A] [answered]
I have tried from last few days is it possible to remove this confirmation box which is coming when switching from react native desktop app?

Because I am not able to handle Yes/No functionality also I am not able to update the content for this box.

![397609_dialog](https://github.com/microsoft/react-native-windows/assets/13333370/93084c7a-7504-47c6-a59a-1864ef49c5de)

Please suggest me if there is any possibility to remove this or i can handle content and Yes/No actions.

**Comment by chrisglein** [comment, marked as answer]
Is this article helpful? https://devblogs.microsoft.com/oldnewthing/20181119-00/?p=100265

**Reply by rahulmishra1991** [reply]
How we can implement this for windows app?

**Reply by chrisglein** [reply]
Can you clarify what specific action/code is triggering this? More context would help.

**Reply by rahulmishra1991** [reply]
Yes if possible then please share how exactly we can integrate into RNW.

**Reply by chrisglein** [reply]
Can you share some code snippets and the overall structure of your app? Do you have a protocol handler that is involved in this?
